music21/musicxml/m21ToXml.py

```diff
@@ -861,7 +861,7 @@ def setColor(self, mxObject, m21Object):
         '''
         Sets mxObject['color'] to a normalized version of m21Object.style.color
         '''
-        self.setStyleAttributes(mxObject, m21Object, 'color')
+        self.setStyleAttributes(mxObject, m21Object, 'color', 'color')
```

Comment:
Not sure why this change?  If the last attribute is omitted it should be the same as the former, no?

Reply:
No, setStyleAttributes runs hyphenToCamelCase on the musicxmlNames if m21Names is None, so we run it 10,000+ times, i.e. once per note. It's not an awful cost, since we stopped compiling a regex every time in an earlier PR, but just seemed like an easy win for heavily used parsers.

Reply:
AH! Makes perfect sense.

Reply:
Maybe add in a notice why you're doing that or I'm libel to forget and remove it later!